workflow/workflow.go

```diff
@@ -368,29 +368,29 @@ func (l *loopState) unresolveOutputs(stepID string, stageID string, skippedOutpu
 				l.logger.Warningf("Could not get DAG node %s (%s)", stepID+"."+stage.ID+"."+stageOutputID, err.Error())
 				continue
 			}
-			l.logger.Debugf("Will unresolve node %s in the DAG", stepID+"."+stage.ID+"."+stageOutputID)
+			l.logger.Debugf("Will resolve node %s in the DAG as unresolvable", stepID+"."+stage.ID+"."+stageOutputID)
```

Comment:
OK, "resolve [...] as unresolvable" kinda bugs me. Why not "mark [...] as unresolvable", matching the function name and comment? Same for the `Errorf` text below, and the pair in `markStageNodeUnresolvable`.